configure.ac

```diff
@@ -1276,6 +1276,14 @@ if test "x${enable_pcre}" != "xyes"; then
   AC_MSG_ERROR([Cannot find pcre library. Configure --with-pcre=DIR])
 fi
 
+#
+# Check for tbb library
+#
+TS_CHECK_TBB
+if test "x${enable_tbb}" != "xyes"; then
+  AC_MSG_ERROR([Cannot find tbb library. Configure --with-tbb=DIR])
```

Comment:
This check shouldn't cause an error but QUIC should be enabled only if TBB is available (same approach as the check for TLS 1.3) until we use it in core.